about.html

```diff
@@ -0,0 +1,138 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <script src="https://kit.fontawesome.com/d2a3ee28df.js" crossorigin="anonymous"></script>
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet"
+    integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
+  <link rel="stylesheet" href="css/about.css">
+  <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Lato&display=swap" rel="stylesheet">
+  <title>About openBIM</title>
+</head>
+
+<body>
+  <div id="navWrapper">
+    <nav class="topnav">
+      <div id="navLinksWrapper">
+        <img src="static/facebook.png" alt="">
+        <img src="static/twitter.png" alt="twitter logo">
+        <a href="#spanish">Spanish</a>
+        <a href="#mypage">My Page</a>
+        <a href="#logout">Logout</a>
+      </div>
+    </nav>
+    <nav class="desktop-nav">
+      <div class="logoAbout">
+        <img src="static/ifclogo.png" alt="facebook logo">
+        <h3>Open BIM Global <br> Summit 2022</h3>
+      </div>
+      <ul class="conta">
+        <li><a href="#about">About</a></li>
+        <li><a href="index.html#program">Program</a></li>
+        <li><a href="https://www.buildingsmart.org/about/openbim/">Join</a></li>
+        <li><a href="index.html#sponsors">Sponsor</a></li>
+        <li><a href="https://www.buildingsmart.org/resources/latest-news/">News</a></li>
+        <li><a id="aButton" href="https://www.buildingsmart.org/chapter-directory/">BIM Campaing</a></li>
```

Comment:
- [ ] Please, consider that there is no possibility to navigate to the Home page from the About page, both desktop and mobile versions.

Reply:
✔️ well resolved